### PR TITLE
docs: Add Ping360 settings

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: docs/device-settings-ping360.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+![Ping360 Settings](/ping-viewer/images/viewer/ping360-settings.png)
+
+## Menu Items
+
+- **Range:** Range control slider, performs the necessary sensor configuration for such distance range.
+- **Receiver Gain:** Adjust sensor sensitivity, between **low**, **medium** and **high**.
+- **Sector Angle:** Changes the scan angle range, helps to increase the scan time in specific areas.
+- **Reset Settings:** Reset sensor configuration, the default values for sample and scan will be used after that.
```
Comment (Williangalvani) [inline docs/device-settings-ping360.md]: what are `sample` and `scan` here?

File: docs/device-settings-ping360.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+#### Advanced Settings
+
+> Click the arrow on the Device Settings menu to access these advanced settings
+
+- **Transmit Duration:** Manual selection of the emitted pulse duration. Can be used to change the power of the reflected signal.
+- **Transmit Frequency:** Changes the frequency of the emitted pulse. Helps to adjust properties based in the signal's frequency.
+- **Speed if Sound:** The speed of sound to be used for distance calculations. This should be 1500 m/s in salt water, 1450 m/s in fresh water. (it's not possible to use Ping360 in air!)
+- **Angular Resolution:** Changes the step between each sample. Can be used to improve the scan in different sectors range with lowers resolution.
```
Comment (Williangalvani) [inline docs/device-settings-ping360.md]:
```suggestion
- **Angular Resolution:** Changes the step between each sample. Enables scanning a sector faster at the expense of a lower angular resolution.
```

File: docs/device-settings-ping360.md (hunk 1 above)
Comment (jaxxzer) [reply docs/device-settings-ping360.md]: "Reset the settings in this panel to their default configuration"

File: docs/device-settings-ping360.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+![Ping360 Settings](/ping-viewer/images/viewer/ping360-settings.png)
+
+## Menu Items
+
+- **Range:** Range control slider, performs the necessary sensor configuration for such distance range.
+- **Receiver Gain:** Adjust sensor sensitivity, between **low**, **medium** and **high**.
+- **Sector Angle:** Changes the scan angle range, helps to increase the scan time in specific areas.
```
[the remark this answers is not in the document]
Comment (jaxxzer) [reply docs/device-settings-ping360.md]: '**Sector Angle:** Choose full 360 scan, or select a smaller area to scan directly in front of the sonar'

File: docs/device-settings-ping360.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+![Ping360 Settings](/ping-viewer/images/viewer/ping360-settings.png)
+
+## Menu Items
+
+- **Range:** Range control slider, performs the necessary sensor configuration for such distance range.
+- **Receiver Gain:** Adjust sensor sensitivity, between **low**, **medium** and **high**.
```
[the remark this answers is not in the document]
Comment (jaxxzer) [reply docs/device-settings-ping360.md]: `**Receiver Gain:** Adjusts receiver sensitivity, choices are **low**, **medium** and **high**`

File: docs/device-settings-ping360.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+![Ping360 Settings](/ping-viewer/images/viewer/ping360-settings.png)
+
+## Menu Items
+
+- **Range:** Range control slider, performs the necessary sensor configuration for such distance range.
```
[the remark this answers is not in the document]
Comment (jaxxzer) [reply docs/device-settings-ping360.md]: `**Range:** adjust the distance from the sonar to scan signals. Smaller ranges will scan faster as the receiver does not have to wait as long to receive a response. Changes to this setting will automatically adjust the transmit duration (advanced setting) for best performance in most applications.`

File: docs/device-settings-ping360.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+- **Receiver Gain:** Adjust sensor sensitivity, between **low**, **medium** and **high**.
+- **Sector Angle:** Changes the scan angle range, helps to increase the scan time in specific areas.
+- **Reset Settings:** Reset sensor configuration, the default values for sample and scan will be used after that.
+
+#### Advanced Settings
+
+> Click the arrow on the Device Settings menu to access these advanced settings
+
+- **Transmit Duration:** Manual selection of the emitted pulse duration. Can be used to change the power of the reflected signal.
+- **Transmit Frequency:** Changes the frequency of the emitted pulse. Helps to adjust properties based in the signal's frequency.
```
Comment (jaxxzer) [inline docs/device-settings-ping360.md]:
```suggestion
- **Transmit Frequency:** control the acoustic frequency.
```

**Comment by patrickelectric** [comment]
Corrections done.

**Pull request merged**